spec/rubocop/cop/lint/format_parameter_mismatch_spec.rb

```diff
@@ -85,6 +85,10 @@
     expect_no_offenses("format('%s %s %% %s %%%% %%%%%%', 1, 2, 3)")
   end
 
+  it 'correctly ignores CGI escaped brackets' do
+    expect_no_offenses("format('%s%%5B%s%%5D=%s', 'name', 'key', 'value')")
```

Comment:
This test case is too complicated.I think `format("%s%%5B", 'name')` is enough.
And the description (`correctly ignores ...`) should be changed because the problem is not limited to CGI. 
WDYT?